EventsExpress/ClientApp/src/actions/event-item-view.js

```diff
@@ -90,6 +90,27 @@ export function join(userId, eventId) {
   }
 }
 
+// ACTION APPROVER FOR USER
+export function approveUser(userId, eventId, ButtonAction) {
```

Comment:
Please follow the camel case naming convention when naming variables and arguments.